templates/20-04/index.html

```diff
@@ -32,6 +32,7 @@
     {%- endif -%}
     {%- if slot == 'cta' -%}
       <a href="/pro" class="p-button--positive">Get started with Ubuntu Pro for 20.04 LTS</a>
+      <a href="/pro/free-trial">Start a free trial</a>
```

Comment:
Please add nbsp and chevron per [webteam best practices for cta's](https://webteam.canonical.com/practices/bau-reviews#sites-tribe-conventions-and-best-practices):

> CTA’s that are not styled as buttons should always have a trailing &nbsp;&rsaquo;

```suggestion
      <a href="/pro/free-trial">Start a free trial&nbsp;&rsaquo;</a>
```